stetho/src/main/java/com/facebook/stetho/inspector/screencast/ScreencastDispatcher.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.screencast;
+
+import android.app.Activity;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Matrix;
+import android.graphics.RectF;
+import android.os.Handler;
+import android.os.HandlerThread;
+import android.os.Looper;
+import android.util.Base64;
+import android.util.Base64OutputStream;
+import android.view.View;
+import com.facebook.stetho.common.LogUtil;
+import com.facebook.stetho.inspector.elements.android.ActivityTracker;
+import com.facebook.stetho.inspector.jsonrpc.JsonRpcPeer;
+import com.facebook.stetho.inspector.protocol.module.Page;
+
+import java.io.ByteArrayOutputStream;
+
+public final class ScreencastDispatcher {
+  private static final float FRAMERATE = 60f;
```

Comment:
Calling this a "framerate" is quite a misnomer since we won't actually be able to compress a PNG/JPEG at 60fps. We're not making an attempt to actually stream at 60 frames per second. Rather, this value is being used to calculate an inter-frame delay. And, having an inter-frame delay of floor(1000/60)= 16 milliseconds is very, very aggressive.

I'd prefer just having the FRAME_DELAY constant and setting it to something that hopefully won't cause low-end devices to spend almost all of their available CPU time on screencast. Maybe try 200ms?